INSTALL.md

```diff
@@ -21,6 +21,7 @@ i.e. when using distro provided Qt make sure you have (or the equivalent for Qt6
 
 - Redhat,Fedora: qt5-qtbase-private-devel
 - Debian,Ubuntu: qtbase5-private-dev qtdeclarative5-private-dev
+qtattributionsscanner-qt5 qhelpgenerator-qt5 qdoc-qt5
```

Comment:
shouldn't this be appended to the previous line?

Reply:
No Markdownlint complains that the line is otherwise too long.

Reply:
Oh, I noticed that the linter was not yet happy. I have added a space.

Reply:
Looks ok in the preview.